config/default.go

```diff
@@ -117,4 +117,9 @@ URI = "127.0.0.1:50071"
 [BroadcastServer]
 Host = "0.0.0.0"
 Port = 61090
+
+[Metrics]
+Host = "0.0.0.0"
+Port = 9091
+Enabled = false
```

Comment:
```suggestion

[Metrics]
URI = ""
```

Reply:
@ARR552 we finally decided to stick with `Host`, `Port` and `Enabled` because we find it more explicit. Also it allows to turn metrics on/off without having to type the `URI` every time. This is helpful especially in this first stage where we are still testing the metrics. WDYT?